src/main/java/com/documents/controllers/StudentController.java

```diff
@@ -29,4 +40,65 @@
             return new ResponseEntity<Student>(HttpStatus.NOT_FOUND);
         }
     }
+
+    /**
+     * Return all the students from database
+     *
+     * @return
+     */
+    @RequestMapping(value = "/student/all", method = RequestMethod.GET)
+    public ResponseEntity<List<Student>> getAllStudents() {
+        List<Student> all = studentService.findAll();
+        return new ResponseEntity<List<Student>>(all, HttpStatus.OK);
+    }
+
+    /**
+     * Delete a specific student from database
+     *
+     * @param id
+     *
+     * @return
+     */
+    @RequestMapping(value = "/student/{id}", method = RequestMethod.DELETE)
+    public ResponseEntity<Student> deleteStudent(@PathVariable Long id) {
+        Student student = studentService.findById(id);
+        if (student != null) {
+            studentService.delete(id);
+        } else {
+            return new ResponseEntity<Student>(HttpStatus.NO_CONTENT);
+        }
+        return new ResponseEntity<Student>(HttpStatus.OK);
+    }
+
+    /**
+     * Insert a new student into the database
```

Comment:
Edit all the comments either by adding comments for all  the params and the retured type, either by deleting them and leaving a simple description for the method.